### PR TITLE
Edit Keycloak 24 release notes

### LINKED ISSUE #27326: Edit Keycloak 24.0 release notes
### Description

Conform style of new features to the existing Keycloak standards

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by andymunro**
Closes #27326
@mposolda, @jonkoops, I am editing the 24 new features list for the upgrading guide so it can be reused for RHBK 24 and improve some grammar.  Can you check it  to verify no meaning is lost?

### CODE REVIEW

**Review by ahus1 — COMMENTED** [review]
Changes LGTM, waiting for feedback from Marek and Jon before merging.

File: docs/documentation/release_notes/topics/24_0_0.adoc [hunk 1]
```diff
@@ -1,34 +1,36 @@
...
+* Migration from previous versions - The user profile is now always enabled, but it operates as before for those who did not use this feature. You can
+benefit from the user profile capabilities, but you are not required to use them. For migration instructions, see the link:{upgradingguide_link}[{upgradingguide_name}].
 
-This first release of the user profile as a supported feature is just the starting point and the baseline for delivering many more capabilities around identity management.
+The first release of the user profile as a supported feature is just the starting point and the baseline for delivering many more capabilities around identity management.
 
-We would like to huge Thanks to awesome Keycloak community as lots of ideas, requirements and contributions came from the community! Special thanks to:
+ifeval::[{project_community}==true]
+We would like to give huge thanks to theawesome Keycloak community as lots of ideas, requirements and contributions came from the community! Special thanks to:
```
Comment (mposolda) [inline docs/documentation/release_notes/topics/24_0_0.adoc:33]:
```suggestion
We would like to give huge thanks to the awesome Keycloak community as lots of ideas, requirements and contributions came from the community! Special thanks to:
```

**Review by jonkoops — APPROVED** [review]

**Review by mposolda — COMMENTED** [review]
@andymunro Looks good. Added one very minor suggestion, which I think is a typo.

Note that I've sent another PR updating release notes https://github.com/keycloak/keycloak/pull/27339 and I think it may have some conflicts with your PR. I am leaving to you whether my PR or your PR should be merged first.

**Review by mposolda — APPROVED** [review]

**Comment by mposolda** [comment]
@jonkoops @ahus1 @andymunro FYI. I've merged the PR despite I found one small typo in it. I will fix this typo in my other PR for the release notes https://github.com/keycloak/keycloak/pull/27339, which I would rebase on top of this one.

**Pull request merged**